src/main/java/io/strimzi/kafka/metrics/PrometheusMetricsReporterConfig.java

```diff
@@ -7,50 +7,87 @@
 import io.prometheus.client.exporter.HTTPServer;
 import org.apache.kafka.common.config.AbstractConfig;
 import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.common.config.ConfigException;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
 import java.net.BindException;
+import java.net.InetSocketAddress;
 import java.util.List;
 import java.util.Map;
+import java.util.Objects;
 import java.util.Optional;
+import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 /**
- * Configuration for the reporter implementations.
+ /**
+ * Configuration for the PrometheusMetricsReporter implementation.
  */
 public class PrometheusMetricsReporterConfig extends AbstractConfig {
 
     private static final Logger LOG = LoggerFactory.getLogger(PrometheusMetricsReporterConfig.class.getName());
 
+    /**
+     * Configuration prefix for Prometheus Metrics Reporter.
+     */
     public static final String CONFIG_PREFIX = "prometheus.metrics.reporter.";
```

Comment:
This field could be private so you don't have to add javadoc

Reply:
Changed to private

Reply:
It is still public

Reply:
CONFIG_PREFIX now private